### ISSUE TITLE
[BUG] 微调时数据集中图片无法正确加载

### ISSUE THREAD

**Issue by Ataraxy33** [QwenLM/Qwen-VL#332]
### 是否已有关于该错误的issue或讨论？ | Is there an existing issue / discussion for this?

- [X] 我已经搜索过已有的issues和讨论 | I have searched the existing issues / discussions

### 该问题是否在FAQ中有解答？ | Is there an existing answer for this in FAQ?

- [X] 我已经搜索过FAQ | I have searched FAQ

### 当前行为 | Current Behavior

当使用脚本LoRa微调Qwen-VL模型时，如果加载的data中有图片路径时，在.cache缓存文件中则会报错如下：
······
File "/home/zm2024/.cache/huggingface/modules/transformers_modules/Qwen-VL-Chat/modeling_qwen.py", line 657, in forward
    hidden_states[i][a + 1 : b] = images[idx]
RuntimeError: a view of a leaf Variable that requires grad is being used in an in-place operation.

如果加载的数据集中没有图片，仅有文本输入，则可以正确开始训练。

### 运行环境 | Environment

```Markdown
- OS: Ubuntu 20.04
- Python: 3.10
- Transformers: 4.32.0
- PyTorch: 2.2.1
- CUDA (`python -c 'import torch; print(torch.version.cuda)'`):12.1
```

**Comment by J0eky** [comment]
hi, have you solved it? @Ataraxy33

**Comment by Ataraxy33** [comment]
> hi, have you solved it? @Ataraxy33

yes, I use another tool to finetune it and it works. Please check this link: [https://github.com/modelscope/swift](https://github.com/modelscope/swift)

**Comment by 1180300419** [comment]
@J0eky Hey，have you solved it？

**Comment by miovovo** [comment]
@Ataraxy33 hello! Your link here seems to be invalid, would you mind to share again? I got the same bug

**Comment by Ataraxy33** [comment]
> @Ataraxy33 hello! Your link here seems to be invalid, would you mind to share again? I got the same bug

Okay, please check this new link: [https://github.com/modelscope/swift](https://github.com/modelscope/swift)

**Comment by wade30822** [comment]
请问下训练集的图片大小和数据量保持在多少比较合适
